HackingWithSwift/SwiftUI/myProject2/myProject2/myProject2/ContentView.swift

```diff
@@ -0,0 +1,201 @@
+                            .background(Color.black)
+                            .clipShape(.buttonBorder)
+                            .font(.system(size: 70))
+                            
+                            Button("✋") {
+                                counter += 1
+                                paper()
+                            }
+                            .padding(20)
+                            .background(.black)
+                            .clipShape(.buttonBorder)
+                            .font(.system(size: 70))
+                            
+                            Button("✌️"){
+                                counter += 1
+                                scissons()
+                            }
+                            .padding(20)
+                            .background(.black)
+                            .clipShape(.buttonBorder)
+                            .font(.system(size: 70))
+                        }
+                    }
+                    
+                    VStack{
+                        Spacer()
+                        Text("Score: \(score)")
+                    }
+                    .font(.system(size: 40))
+                }
+            }
+            .alert("Game over", isPresented: $endGame) {
+                Button ("Restart") {}
+            } message: {
+                Text("\(finalScore)")
+            }
+            .navigationTitle("Rock, Paper, Scissons!")
+        }
+    }
+    
+    func compChoose() -> some View {
+        Text(optionToСhoice[computerСhoice])
+            .font(.largeTitle.weight(.semibold))
+    }
+    
+    func rock()  {
+        if computerСhoice == 0 {
+            myChoise = "✊"
+            score += 1
+            newGame()
+            rightСhoice = true
+        }
+        else if computerСhoice == 1 {
+            myChoise = "✊"
+            score -= 1
+            newGame()
+        } else {
+            myChoise = "✊"
+            score += 1
+            newGame()
+            rightСhoice = true
+        }
+    }
+    
+    func paper() {
+        if computerСhoice == 0 {
+            myChoise = "✋"
+            score += 1
+            newGame()
+            rightСhoice = true
+        }
+        else if computerСhoice == 1 {
+            myChoise = "✋"
+            score += 1
+            newGame()
+            rightСhoice = true
+        } else {
+            myChoise = "✋"
+            score -= 1
+            newGame()
+        }
+    }
+    
+    func scissons() {
+        if computerСhoice == 0 {
+            myChoise = "✌️"
+            score -= 1
+            newGame()
+        }
+        else if computerСhoice == 1 {
+            myChoise = "✌️"
+            score += 1
+            newGame()
+            rightСhoice = true
+        } else {
+            myChoise = "✌️"
+            score += 1
+            newGame()
+            rightСhoice = true
+        }
+    }
+    
+    func myturn() -> some View {
```

Comment:
Functions that return UI components should follow a naming convention like myTurnLabel or getMorePointsButton. The name should start with descriptive words about the function’s purpose, and end with the type of component it returns, such as Label, Button, or GroupOfLabels etc, to clearly indicate what it produces.